.gitignore

```diff
@@ -81,6 +81,7 @@ build
 
 #fastlane
 /fastlane/report.xml
+/fastlane/metadata/
```

Comment:
This folder is generated in the way?

Reply:
Yes in this folder store the screenshot and all the meta data right now this folder is stored like artifact on CircleCI